AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m

```diff
@@ -0,0 +1,31 @@
+#import <Foundation/Foundation.h>
```

Comment:
Probably not necessary. It is already imported in header file.

Reply:
Got it, removed.